SweetsFactory/package.json

```diff
@@ -20,7 +20,7 @@
     "core-js": "^2.5.0",
     "firebase": "^4.2.0",
     "foundation-sites": "^6.4.4-rc1",
-    "jquery": "^3.2.1",
+    "jquery": "^3.2.0",
```

Comment:
Which was the reason for changing the jQuery version?

Reply:
I had some issues with using $, i don't think that was the fix to that but i left it that way, but finally worked after an npm instal